documentation.md

```diff
@@ -0,0 +1,233 @@
+# Cyberwatch API Toolbox documentation
+
+## Available methods
+
+#### Ping
+
+Send a GET request to `/api/v2/ping` to validate that the API is working.
+
+###### Usage example and expected result:
+
+```python
+>>> CBWApi(URL, API_KEY, SECRET_KEY).ping()
+INFO:root:OK
+True
+```
+
+#### Servers
+
+Send a GET request to `/api/v2/servers` to retrieve the list of all servers.
+
+###### Usage example and expected result:
+
+```python
+>>> CBWApi(URL, API_KEY, SECRET_KEY).servers()
+[<cbw_api_toolbox.cbw_objects.cbw_server.CBWServer ...]
+```
+
+#### Server
+
+Send a GET request to `/api/v2/servers/{SERVER_ID}` to retrieve the information of a server.
+
+##### Usage example and expected result:
+
+```python
+>>> CBWApi(URL, API_KEY, SECRET_KEY).server(SERVER_ID)
+[<cbw_api_toolbox.cbw_objects.cbw_server.CBWServer]
+```
+
+#### Update server
+
+Send a PATCH request  `/api/v2/servers/{SERVER_ID}` to update the information of a server.
+
+```python
+>>> CBWApi(URL, API_KEY, SECRET_KEY).update_server(SERVER_ID, INFO)
+True
+```
+
+#### Delete server
+
+Send a DELETE request `/api/v2/servers/{SERVER_ID}` to delete SERVER_ID.
```

Comment:
We delete a server not a `SERVER_ID` (same comment as the previous review)

=> You use the ID of a Server (called SERVER_ID) to find the server you want to delete....